envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/routes/EnvoyIngressRoutesFactory.kt

```diff
@@ -34,13 +36,33 @@ class EnvoyIngressRoutesFactory(
             .setIdleTimeout(timeoutIdle)
     }
 
-    private val statusPathPrefix = properties.routes.status.pathPrefix
+    private val statusEndpointsMatch: List<EndpointMatch> = properties.routes.status.endpoints
 
     private val endpointsPrefixMatcher = HeaderMatcher.newBuilder()
             .setName(":path").setPrefixMatch("/").build()
 
-    private val statusPrefixMatcher = HeaderMatcher.newBuilder()
-            .setName(":path").setPrefixMatch(statusPathPrefix).build()
+    private val statusPrefixMatcher: List<HeaderMatcher> = statusEndpointsMatch.map {
```

Comment:
The name `statusPrefixMatcher` is no longer accurate. I would change it to `statusMatcher`

Reply:
done